### PR TITLE
Added lecture_3

### PR DESCRIPTION
**Pull request by Babajko**
Added lecture_3

### CODE REVIEW

File: lecture_3/Exercise_3/main.c [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    node_t * current = head;
+
+    while (current != NULL) 
+    {
+        printf("%d\n", current->val);
+        current = current->next;
+    }
+}
+
+int listRemoveLast(node_t * head) {
```
Comment (rikorsev) [inline lecture_3/Exercise_3/main.c]: I would prefer to have a function that allows to remove some item in the middle of a list, not just last element.

File: lecture_3/Exercise_3/main.c [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+typedef struct node 
+{
+    int val;
+    struct node * next;
+} node_t;
+
+
+void listAdd(const node_t * head, int val) 
+{
+    node_t * current = head++;
```
[the remark this answers is not in the document]
Comment (Babajko) [reply lecture_3/Exercise_3/main.c]: you right it is mistake.

File: lecture_3/Exercise_3/main.c (hunk 1 above)
Comment (Babajko) [reply lecture_3/Exercise_3/main.c]: Yes,  I first time added this function but when i was doing review this code i deleted one.

**Comment by Babajko** [comment]
updated according to comments

**Pull request closed without merging**